drivers/docker/config.go

```diff
@@ -276,7 +276,7 @@ var (
 			"volume_options": hclspec.NewBlock("volume_options", false, hclspec.NewObject(map[string]*hclspec.Spec{
 				"no_copy": hclspec.NewAttr("no_copy", "bool", false),
 				"labels":  hclspec.NewBlockAttrs("labels", "string", false),
-				"driver_config": hclspec.NewBlockSet("driver_config", hclspec.NewObject(map[string]*hclspec.Spec{
+				"driver_config": hclspec.NewBlock("driver_config", false, hclspec.NewObject(map[string]*hclspec.Spec{
```

Comment:
This was a bug fix.  The build of the first sha of this PR fails in [1], before the fix with:

```
=== FAIL: drivers/docker TestConfig_ParseAllHCL (0.00s)
    testing.go:74: 
        	Error Trace:	testing.go:74
        	            				config_test.go:298
        	Error:      	Received unexpected error:
        	            	[pos 1321]: readContainerLen: Unrecognized descriptor byte: hex: 82, decimal: 130
        	Test:       	TestConfig_ParseAllHCL
```

[1] https://travis-ci.org/hashicorp/nomad/jobs/491951636